azure-pipelines.yml

```
@@ -39,3 +39,14 @@ jobs:
   parameters:
     agentOs: Linux
     configuration: Release
+
+- job: Helix
+  pool:
+    name: Hosted VS2017
+    vmImage: vs2017-win2016
+  steps:
+  - checkout: self
+    clean: true
+  - script: .\build.cmd -ci /t:Helix /p:Configuration=Debug
```

```suggestion
  - script: .\build.cmd -ci /t:Helix
```

We usually don't run the Debug config for tests. Let's try doing this for Release configs instead.